src/js/consts/applicationUrls.js

```diff
@@ -129,8 +129,9 @@ const CATEGORY_URL = {
 
 const PRODUCT_SUPPLIER_URL = {
   base: `${CONTEXT_PATH}/productSupplier`,
+  list: () => `${PRODUCT_SUPPLIER_URL.base}/list`,
   create: () => `${PRODUCT_SUPPLIER_URL.base}/create`,
-  edit: (id) => `${PRODUCT_SUPPLIER_URL.base}/edit/${id}`,
+  edit: (id) => `${PRODUCT_SUPPLIER_URL.base}/create/${id}`,
```

Comment:
maybe let's do it later (at the end), so we can still edit product sources if we wanted to test something (assuming we click the Edit from the list page)

Reply:
you can still access the old page manually

Reply:
I know we can, this is why I wrote this: "(assuming we click the Edit from the list page)".
But let's leave it, as accessing edit through show page should not be "frustrating"